Tiles/Furniture/Monoliths/PurpleDistortedMonolithTile.cs

```diff
@@ -37,6 +38,8 @@ public override void SetStaticDefaults()
             AnimationFrameHeight = 92;
         }
 
+        public override string HighlightTexture => "CalamityMod/Tiles/Furniture/Monoliths/BlueDistortedMonolithTile_Highlight";
```

Comment:
As a note, since you shared a highlight here instead of copying the asset, respriting the Purple Distorted Monolith will have a gotcha / extra step.